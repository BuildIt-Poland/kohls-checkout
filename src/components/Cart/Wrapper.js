import styled from 'styled-components';

const Wrapper = styled.article`
  padding-bottom: 12rem;
`;

export default Wrapper;
