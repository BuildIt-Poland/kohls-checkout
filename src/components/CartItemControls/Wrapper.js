import styled from 'styled-components';

import { spacingMedium, demoYellow } from '../../styles/designTokens';

const Wrapper = styled.div`
  background: ${demoYellow};
  display: flex;
  justify-content: space-between;
  margin-top: ${spacingMedium};
`;

export default Wrapper;
