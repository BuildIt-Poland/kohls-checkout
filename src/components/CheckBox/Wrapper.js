import styled from 'styled-components';

import { spacingLarge, spacingSmall } from '../../styles/designTokens';

const Wrapper = styled.div`
  position: relative;
  display: block;
  padding-left: ${spacingLarge};
  margin: ${spacingSmall};
  cursor: pointer;
`;

export default Wrapper;
