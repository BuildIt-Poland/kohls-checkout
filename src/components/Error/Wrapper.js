import styled from 'styled-components';

import {
  colorRed,
  colorLightRed,
  colorBlack,
  fontMedium,
  fontWeightBold,
  spacingSmall,
  borderWidthThick
} from '../../styles/designTokens';

// TODO: Probably add generic InfoBox later @blurbyte
const Wrapper = styled.p`
  width: 100%;
  border: ${borderWidthThick} solid ${colorRed};
  color: ${colorBlack};
  background-color: ${colorLightRed};
  font-size: ${fontMedium};
  font-weight: ${fontWeightBold};
  padding: ${spacingSmall};
`;

export default Wrapper;
