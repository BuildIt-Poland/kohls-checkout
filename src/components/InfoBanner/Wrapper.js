import styled from 'styled-components';
import { colorTurquoise, colorWhite, fontMedium, spacingSmall } from '../../styles/designTokens';

const Wrapper = styled.section`
  display: flex;
  justify-content: center;
  padding: ${spacingSmall} 0;
  background-color: ${colorTurquoise};
  color: ${colorWhite};
  font-size: ${fontMedium};
  text-align: center;
`;

export default Wrapper;
