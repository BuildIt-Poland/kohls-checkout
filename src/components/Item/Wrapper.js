import styled from 'styled-components';

const Wrapper = styled.div`
  display: grid;
  grid-template-columns: 30% auto;
  align-items: center;
  margin: 3rem 1.3rem;
`;

export default Wrapper;
