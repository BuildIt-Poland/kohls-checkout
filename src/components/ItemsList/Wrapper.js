import styled from 'styled-components';

import { colorLightGrey, borderWidthThin, spacingLarge } from '../../styles/designTokens';

const Wrapper = styled.section`
  margin-top: ${spacingLarge};
  border-top: ${borderWidthThin} solid ${colorLightGrey};
  padding-top: ${spacingLarge};
`;

export default Wrapper;
