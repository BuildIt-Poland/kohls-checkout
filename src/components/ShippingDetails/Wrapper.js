import styled from 'styled-components';

import { spacingLarge, spacingMedium } from '../../styles/designTokens';

const Wrapper = styled.div`
  display: grid;
  grid-template-columns: 6rem 1fr;
  grid-gap: ${spacingMedium};
  margin-bottom: ${spacingLarge};
  align-items: center;
`;

export default Wrapper;
