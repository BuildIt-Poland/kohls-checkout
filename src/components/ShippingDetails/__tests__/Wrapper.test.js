import React from 'react';
import { create } from 'react-test-renderer';

import Wrapper from '../Wrapper';

describe('COMPONENT - ShippingDetails Wrapper', () => {
  it('renders correctly', () => {
    const component = create(<Wrapper />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
