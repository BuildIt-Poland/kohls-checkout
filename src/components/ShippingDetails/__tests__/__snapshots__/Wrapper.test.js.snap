// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - ShippingDetails Wrapper renders correctly 1`] = `
.c0 {
  display: grid;
  grid-template-columns: 6rem 1fr;
  grid-gap: 2rem;
  margin-bottom: 3rem;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

<div
  className="c0"
/>
`;
